src/librustc/dep_graph/dep_node.rs

```diff
@@ -35,7 +35,7 @@
 //! "infer" some properties for each kind of `DepNode`:
 //!
 //! * Whether a `DepNode` of a given kind has any parameters at all. Some
-//!   `DepNode`s, like `AllLocalTraitImpls`, represent global concepts with only one value.
+//!   `DepNode`s, like `?`, represent global concepts with only one value.
```

Comment:
Is this supposed to be a  `?`?

Reply:
No, but this PR removed all the suitable examples...

Reply:
I've removed the `?` and instead just don't use an example here.